templates/page.html

```diff
@@ -282,12 +282,18 @@ <h6>released on 23 April 2018</h6>
         <div class="events-list-container">
           <table class="events-list">
             <tbody>
+                <tr>
+                  <td class="events-list-header"><i class="fab fa-slideshare event-conference"></i></td>
+                  <td>
+                    We shall participate at the 14th eScience IEEE International Conference, 29 October - 1 November 2018, Amsterdam, the Netherlands. Meet you
+    there! <a href="https://www.escience2018.com/">[conference]</a>
+                  </td>
+                </tr>
               <tr>
                 <td class="events-list-header"><i class="fab fa-slideshare event-conference"></i></td>
                 <td>
-                  We shall participate at the CHEP2018 Conference on Computing in High
- Energy and Nuclear Physics, 9-13 July 2018, Sofia, Bulgaria. Meet you
- there! <a href="http://chep2018.org/">[conference]</a>
+                  REANA presented at CHEP2018 Conference on Computing in High
+ Energy and Nuclear Physics, 9-13 July 2018, Sofia, Bulgaria. <a href="https://indico.cern.ch/event/587955/contributions/2937551/attachments/1679266/2705194/reana-chep2018-20180702.pdf">[slides]</a> <a href="https://indico.cern.ch/event/587955/">[event]</a>
                 </td>
```

Comment:
Can you remove the 1 or 2 old items to make the height shorter?  And use the "P" icon for CHEP2018, since there are slides now...